src/dex-ui-components/presets/AddLiquidityForm/AddLiquidityForm.tsx

```diff
@@ -71,11 +74,32 @@ export function AddLiquidityForm(props: AddLiquidityFormProps) {
   usePoolsData(REFRESH_INTERVAL);
   useSwapData(selectedPairContractId, REFRESH_INTERVAL);
 
+  function createTransactionDeadlineErrorMessage(transactionDeadline: number): string {
```

Comment:
This logic is duplicated across the Swap and Add Liquidity forms. Moving this to the `useFormSettings` hook in the next PR.